src/yang/ietf-qos-action.yang

```diff
@@ -346,13 +346,13 @@ module ietf-qos-action {
       leaf count-action {
         type empty;
         description
-          "Take an action if this is defined.";
+          "Count action";
```

Comment:
I know I suggested the above change, but saying "count action" does not mean much either. Can you be more descriptive?